src/ng/parse.js

```diff
@@ -1763,6 +1768,24 @@ function $ParseProvider() {
     return this;
   };
 
+    /**
+   * @ngdoc method
```

Comment:
Since this is supposed to be private API, we don't want it to appear in the public docs.